templates/permission_require.php

```diff
@@ -0,0 +1,20 @@
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="Content-Language" content="en-GB">
+  <meta name="robots" content="noindex, nofollow">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>iitpConnect</title>
+  <style>body{margin:0;padding:0;font:14px / 18px sans-serif;color:#555;background-color:transparent}html{background:#f1f1f1;background:-moz-radial-gradient(center,ellipse cover,rgba(241,241,241,1) 0,rgba(58,146,200,1) 100%);background:-webkit-radial-gradient(center,ellipse cover,rgba(241,241,241,1) 0,rgba(58,146,200,1) 100%);background:radial-gradient(ellipse at center,rgba(241,241,241,1) 0,rgba(58,146,200,1) 100%);background-repeat:no-repeat;background-attachment:fixed}ol,ul{padding:0;margin:0;list-style:none}a{color:#0084b4;text-decoration:none;outline:0}a:hover,a:focus{text-decoration:underline}p a{line-height:inherit}.container{display:flex;flex-direction:column;justify-content:center;align-items:center;position:relative;margin:0 auto;width:100%;height:100vh;overflow:hidden}.alert-main{display:block;position:relative;background:#fff;border:1px solid rgba(0,0,0,0.1);border-radius:5px;padding:20px 60px;margin:0 20px;box-shadow:0 0 10px rgba(0,0,0,0.05)}svg{position:absolute;bottom:-120px;right:-70px;width:400px;transform:rotate(10deg);z-index:-1}h1,p{position:relative;z-index:10;text-align:center;text-rendering:optimizeLegibility}h1{margin:18px 0 0;font-size:40px;font-weight:200;line-height:1;text-shadow:0 1px 2px rgba(0,0,0,.2)}p,label{margin:10px 0 20px;font-size:18px;font-weight:300;line-height:25px;color:#777}p a{font-weight:bold;color:#1c3d5c}.link-help{padding:.4rem .85rem;font-size:1rem;font-weight:normal;border-radius:.25rem;text-decoration:none;background-color:#f5f5f5;border:1px solid rgba(0,0,0,0.1)}.link-help:hover{background-color:#eee;text-decoration:none}.footer{margin:8px 20px;text-align:left;font-size:11px}.footer ul{margin-bottom:5px}.footer li{display:inline;margin:0 5px;line-height:20px}.footer li,.footer a{color:#1c3d5c}.footer a:hover{color:#59b0ff}.links{display:block;text-align:center;margin-top:4rem;margin-left:auto;margin-right:auto;font-size:1rem}.links li{display:inline-block;margin-top:20px}@media screen and (max-width:480px){.container{height:auto;padding-top:20px;padding-bottom:20px}h1{font-size:30px}.link-help{white-space:nowrap}}</style><!-- Sets the page styling, IMPORTANT DO NOT REMOVE -->
+</head>
+<body >
+<div class="container">
+  <div class="container-main">
+    <div class="alert-main">
+      <h1 id="headerText">Sorry, your folder Does not have Full Permission to access this app</h1>
+      <p><span id="descText1">Your host needs to change the Permission from {{permission}} to Minimum 755.</span></p>
```

Comment:
Improve message `>Sorry, your folder does not have minimum permission to run this application.` or something.